modules/virt-cnv-virt-alerts.adoc

```diff
@@ -0,0 +1,487 @@
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get kubevirt -A -o custom-columns="":.metadata.namespace)"
+
+oc -n $NAMESPACE get deploy virt-operator -o yaml
+
+oc -n $NAMESPACE describe deploy virt-operator
+----
+
+. Check if there are issues with the nodes for the control plane and masters, such as if they are in a `NotReady` state.
++
+[source,terminal]
+----
+oc get nodes
+----
+
+.Resolution
+
+There are several reasons for no virt-operator pods being in a `Ready` state. Identify the root cause and take appropriate action.
+
+Otherwise, open a support issue and provide the information gathered in the troubleshooting process.
+
+//VirtAPIDown Alert
+[id="VirtAPIDown_{context}"]
+== VirtAPIDown alert
+
+.Description
+
+All OpenShift API servers are down.
+
+.Reason
+
+If all OpenShift API servers are down, then no API calls for OpenShift entities occur.
+
+.Troubleshoot
+
+. Modify the environment variable `NAMESPACE`.
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get kubevirt -A -o custom-columns="":.metadata.namespace)"
+----
+
+. Verify that there are any running virt-api pods.
++
+[source,terminal]
+----
+oc -n $NAMESPACE get pods -l kubevirt.io=virt-api
+----
+
+.Resolution
+
+Virt-api pods can be down for several reasons. Identify the root cause and take appropriate action.
+
+. Check the status of the virt-api deployment. Use these commands to learn about related events and show if there are any issues with pulling an image, a crashing pod, or other similar problems.
++
+[source,terminal]
+----
+oc -n $NAMESPACE get deploy virt-api -o yaml
+
+oc -n $NAMESPACE describe deploy virt-api
+----
+
+. Check if there are issues with the nodes, such as if the nodes are in a `NotReady` state.
++
+[source,terminal]
+----
+oc get nodes
+----
+
+//VirtApiRESTErrorsBurst Alert
+[id="VirtApiRESTErrorsBurst_{context}"]
+== VirtApiRESTErrorsBurst alert
+
+.Description
+
+More than 80% of the REST calls fail in virt-api in the last five minutes.
+
+.Reason
+
+A very high rate of failed REST calls to virt-api causes slow response, slow execution of API calls, or even complete dismissal of API calls.
+
+.Troubleshoot
+
+. Modify the environment variable `NAMESPACE`.
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get kubevirt -A -o custom-columns="":.metadata.namespace)"
+----
+
+. Check to see how many running virt-api pods exist.
++
+[source,terminal]
+----
+oc -n $NAMESPACE get pods -l kubevirt.io=virt-api
+----
+
+. View the pods' logs using `oc logs` and pod status using `oc describe`.
+
+.Resolution
+
+Virt-api pods can be down for several reasons. Identify the root cause and take appropriate action.
+
+. Check the status of the virt-api deployment to find out more information. These commands provide the associated events and show if there are any issues with pulling an image or a crashing pod.
```

Comment:
It would be good to include, what all information should be collected in case a support ticket is deemed necessary.